client/sites-dashboard/utils.ts

```diff
@@ -66,6 +66,15 @@ export const isMigrationTrialSite = ( site: SiteExcerptNetworkData ) => {
 	return site?.plan?.product_slug === PLAN_MIGRATION_TRIAL_MONTHLY;
 };
 
+export const isMigrationInProgress = ( site: SiteExcerptNetworkData ): boolean => {
+	const migrationStatus = site?.site_migration?.migration_status;
```

Comment:
I saw in the GH Actions that it's reporting an issue here because `SiteExcerptNetworkData` doesn't have the `site_migration`.

Reply:
Thanks! I missed that one. It's fixed in b46936c58a760fe4dc58c931d81b78c0e982440a.